.github/workflows/release.yml

```diff
@@ -0,0 +1,270 @@
+
+env:
+  DOTNET_VERSION: 8.0.x
+  CONFIGURATION: Release
+  DIST_PATH: /tmp/dist
+  OUTPUT_PATH: /tmp/out
+
+jobs:
+  build-leveldb:
+    name: Build leveldb (windows-latest)
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch: [x64, arm64]
+
+    steps:
+    - name: Lookup Cache Distribution
+      id: cache-leveldb
+      uses: actions/cache@v4
+      with:
+        path: ./leveldb/build/Release/*
+        key: leveldb-windows-${{ matrix.arch }}
+        enableCrossOsArchive: true
+        lookup-only: true
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Checkout Repository Code (leveldb)
+      uses: actions/checkout@v4
+      with:
+        repository: google/leveldb
+        path: leveldb
+        submodules: true
+        fetch-depth: 0
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v2
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Setup LevelDb
+      working-directory: ./leveldb
+      run: mkdir -p ./build/Release
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Create Build Files (win-${{ matrix.arch }})
+      working-directory: ./leveldb/build
+      run: cmake -DBUILD_SHARED_LIBS=ON -A ${{ matrix.arch }} ..
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Build (MSBuild)
+      working-directory: ./leveldb/build
+      run: msbuild ./leveldb.sln /p:Configuration=Release
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
+      name: Cache Distribution
+      uses: actions/cache/save@v4
+      with:
+        path: ./leveldb/build/Release/*
+        key: leveldb-windows-${{ matrix.arch }}
+        enableCrossOsArchive: true
+
+  build-neo-cli:
+    needs: [build-leveldb]
+    name: Build & Publish TarBalls (${{ matrix.runtime }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        runtime: [linux-x64, linux-arm64, win-x64, win-arm64, osx-x64, osx-arm64]
+
+    steps:
+    - name: Set Application Version (Environment Variable)
+      run: |
+        APP_VERSION=$(echo '${{ github.event.release.tag_name }}' | cut -d 'v' -f 2)
+        echo "APP_VERSION=$APP_VERSION" >> $GITHUB_ENV
+
+    - name: Checkout Repository Code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+
+    - name: .NET Publish (neo-cli)
+      run: |
+        dotnet publish ./src/Neo.CLI \
+        --version-suffix ${{ matrix.runtime }} \
+        --framework net8.0 \
+        --configuration ${{ env.CONFIGURATION }} \
+        --runtime ${{ matrix.runtime }} \
+        --self-contained true \
+        --output ${{ env.OUTPUT_PATH }}/${{ matrix.runtime }} \
+        --verbosity normal \
+        -p:VersionPrefix=${{ env.APP_VERSION }} \
```

Comment:
too much parameters as well..

Reply:
Too much parameter? Like those are not necessary?

Reply:
I think that a README or comments are necessary, there are several details

Reply:
These are all the settings i found to work.